internal/provider/provider.go

```diff
@@ -61,8 +64,13 @@ func (p *PCBeProvider) Schema(
 					"host": schema.StringAttribute{
 						Required: true,
 					},
+					// one of auth_url or token is required
```

Comment:
This can be enforced with the `"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"` package 

E.g. 

```go
"auth_url": schema.StringAttribute{
    Optional: true,
    Validators: []validator.String{
	    stringvalidator.ConflictsWith(path.MatchRoot("token")),
    },
},
```

Reply:
Cool. I'll check those out, thanks.